resource-rating-plugin-functionality/resource-rating-plugin-user-view.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Plugin Name: Resource Rating
```

Comment:
Please remove the comments, as this should only be included in the main file which is `resource-rating-plugin.php`